source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -17,31 +17,42 @@ The global vulnerability database is created automatically, currently pulling da
-- Full scan: In this scan type, every single package installed as well as the operating system are scanned. It runs only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information. As a result, the newly detected vulnerabilities and the ones that no longer affect the agent will be alerted.
-- Partial scans: Only new packages are scanned and the vulnerabilities detected are alerted.
+For more details we have then three different scan types that are described below.
+
+- `Baseline`: This scan type will be triggered the first time after `Vulnerability Detector` is enabled. It performs a full scan for every single package installed as well as the operating system. As a result, the CVEs inventory will have the information of the vulnerabilities detected but no alerts will be generated.
+- `Full scan`: In this scan type, every single package installed as well as the operating system are scanned. It runs only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information. As a result, the newly detected vulnerabilities and the ones that no longer affect the agent will be alerted.
+- `Partial scan`: Only new packages are scanned and the new vulnerabilities detected are alerted together with deleted or updated packages.
 
 There are few considerations that arise from this behavior:
 
-- The user can not trigger a full scan manually. The only option is to disable and enable Vulnerability Detector to trigger the baseline scan. For that purpose, a service restart is needed.
-- The partial scans generate alerts for new packages, but they do not delete alerts for removed packages.
-- Partial scans can be triggered with a Manager restart.
+- The user can not trigger a `Full scan` manually. The only option is to disable and enable `Vulnerability Detector` to trigger the `Baseline` scan. For that purpose, a service restart is needed.
+- The `partial scan` can be triggered with a manager restart and generate alerts for new packages, as well as alerts for removed packages.
+- The new :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` setting, will protect the manager performance from running too often `Full scans` in case of receiving many updates of vulnerabilities feeds.
+
+Alerts
+^^^^^^
+
+- No alerts are generated for the results of a `Baseline` scan.
+- Detection alerts are generated in both, `Partial scan` and `Full scan` when is detected that a vulnerability affects one of the packages scanned.
+- Removal alerts are generated in both, `Partial scan` and `Full scan`, when is detected that a vulnerability is obsolete.
```

Comment:
Here you are mentioning the status `obsolete` of the vulnerabilities. This is not explained in the documentation and it should be because this information, as well as the type, will be returned in the API requests as well as displayed in the Wazuh UI.

Reply:
These three conclusions about "Alerts" seem to be redundant.
I think that the information under "There are few considerations that arise from this behavior" already cover these facts.

Reply:
I think we can replace the "obsolete" expression, which is an internal status of the vuln_cves table, with "deleted/updated".

Reply:
I agree about the "obsolete" term, I was not completely sure about deleting this block, but it's true that could be redundant in some parts. I still think it should be good idea to have an "Alerts" section.